src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -100,16 +101,15 @@ public RoutingResponse route() {
     var routingErrors = Collections.synchronizedSet(new HashSet<RoutingError>());
 
     if (OTPFeature.ParallelRouting.isOn()) {
+      List<Callable<Object>> tasks = List.of(
+        () -> routeDirectStreet(itineraries, routingErrors),
+        () -> routeDirectFlex(itineraries, routingErrors),
+        () -> routeTransit(itineraries, routingErrors)
+      );
       try {
-        CompletableFuture
-          .allOf(
-            CompletableFuture.runAsync(() -> routeDirectStreet(itineraries, routingErrors)),
```

Comment:
Do you happen to know which executors the CompletableFuture uses by default, @vpaturet ?

Reply:
They use ForkJoinPool.commonPool()
https://dzone.com/articles/be-aware-of-forkjoinpoolcommonpool